parameters.cpp

```diff
@@ -217,8 +217,8 @@ bool Parameters::addParameters(){
    
 // Output variable parameters
    // NOTE Do not remove the : before the list of variable names as this is parsed by tools/check_vlasiator_cfg.sh
+   Readparameters::addComposing("variables.output", "List of data reduction operators (DROs) to add to the grid file output.  Each variable to be added has to be on a new line output = XXX.  Available (20190508): B fg_B BackgroundB vg_BackgroundB fg_BackgroundB PerturbedB vg_PerturbedB fg_PerturbedB E fg_E Rhom vg_Rhom fg_Rhom Rhoq vg_Rhoq fg_Rhoq populations_Rho V vg_V fg_V populations_V populations_moments populations_EnergyDensity_Backstream populations_moments_NonBackstream populations_EffectiveSparsityThreshold populations_RhoLossAdjust LBweight MaxVdt MaxRdt populations_MaxVdt populations_MaxRdt MaxFieldsdt MPIrank vg_rank FsGridRank fg_rank FsGridBoundaryType BoundaryType vg_BoundaryType fg_BoundaryType BoundaryLayer vg_BoundaryLayer fg_BoundaryLayer populations_Blocks fSaved populations_accSubcycles VolE vg_VolE fg_VolE HallE GradPeE VolB vg_VolB fg_VolB BackgroundVolB PerturbedVolB Pressure vg_Pressure fg_Pressure populations_PTensor derivs BVOLderivs GridCoordinates Potential BackgroundVolE ChargeDensity PotentialError MeshData");
```

Comment:
This does not match what is coded in datareducer.cpp. Seems to have slipped into the preceding entry between the name and _Backstream there.